NuGet.Config

```diff
@@ -4,5 +4,8 @@
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
     <add key="NuGet" value="https://nuget.org/api/v2/" />
     <add key="AzureAD" value="http://www.myget.org/F/azureadwebstacknightly"/>
+    <add key="aspnet-contrib" value="https://www.myget.org/F/aspnet-contrib/" />
+    <add key="Roslyn" value="https://www.myget.org/F/roslyn-nightly/" />
```

Comment:
These changes have nothing to do in this PR but are required to make CI happy.
I opened a ticket to track this omission (https://github.com/aspnet/Security/issues/399) and I'll remove them before squashing.